run_checker_scripts/eddie_comparison.py

```diff
@@ -0,0 +1,1861 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
There's a lot of code here - let's put aside some time to talk through this code and treat it more as a handover